app/assets/javascripts/validations.js

```diff
@@ -0,0 +1,67 @@
+$(document).ready(function() {
+    //alert("called");
+    $('#fieldContainerForm').bootstrapValidator({
+        feedbackIcons: {
+            valid: 'glyphicon glyphicon-ok',
+            invalid: 'glyphicon glyphicon-remove',
+            validating: 'glyphicon glyphicon-refresh'
+        },
+        fields: {
+            'user[email]': {
+                //container: '#user[email]Message',
+                validators: {
+                    notEmpty: {
+                        message: 'The email is required and cannot be empty'
+                    }
+                }
+            },
+            'user[password]': {
+                //container: '#user[password]Message',
+                validators: {
+                    notEmpty: {
+                        message: 'The password is required and cannot be empty'
```

Comment:
message: Change from 'The password is required and cannot be empty'  => 'The password is required'
And repeat this process for other messages as well.

Reply:
Done